lib/measured/rails/active_record.rb

```diff
@@ -1,9 +1,71 @@
-module Measured::Rails
+module Measured::Rails::ActiveRecord
   extend ActiveSupport::Concern
 
   module ClassMethods
-    def measured
+    def measured(measured_class, *fields)
+      options = fields.extract_options!
+      options = {}.merge(options)
```

Comment:
Yes, this is redundant for now. But in that hash is where default options can/will be added. Note the direction of the merge.